docs/Deployment/realtime-analytics.md

```diff
@@ -0,0 +1,88 @@
+
+> Refer to [Analytics Realtime](https://github.com/epam/ai-dial-analytics-realtime) repository to learn more and view the project source code.
+
+Analytics Realtime tool is a sink of [vector.dev](https://vector.dev/). It does not retain any private information, such as user prompts or conversations, beyond the system. Instead, only the computed artifacts are collected and stored in time-series databases like InfluxDB or any scalable database capable of handling voluminous, constantly changing information.
+
+Examples of the computed artifacts:
+
+* Who has used the AI? – user hash, title, and never personal data such as names.
+* What areas have people asked questions about?
+* Are there any recurring patterns?
+* Topics of conversations.
+* Unique users.
+* Sentiments.
+* Cost analysis of the communication.
+* Language of conversations.
+* Any other calculated statistics based on conversations.
+
+
+**TODO**
+Update AI DIAL Analytics Realtime [README](https://github.com/epam/ai-dial-analytics-realtime/blob/development/README.md) with a link to this document.
+
+## Configuration
+
+This section outlines the required steps for configuring Analytics Realtime service.
+
+- Step 1: Configure [DIAL Core](https://github.com/epam/ai-dial-core)
+- Step 2: Install [Influx DB](https://github.com/influxdata/influxdb)
+- Step 3: Configure [DIAL Analytics Realtime](https://github.com/epam/ai-dial-analytics-realtime)
+- Step 4: Configure [Prompt Log Collector](https://github.com/vectordotdev/vector)
+- Step 5: Configure [Grafana](https://github.com/grafana/grafana)
+
+### Step 1: AI DIAL Core
+
+Use the default AI DIAL Core [Gflog Configuration](https://github.com/epam/ai-dial-core/blob/development/src/main/resources/gflog.xml) as reference.
+
+### Step 2: Influx DB
+
+Refer to InfluxDB documentation to learn how to [install](https://docs.influxdata.com/influxdb/v2/install/) it and how to [create tokens](https://docs.influxdata.com/influxdb/v2/admin/tokens/create-token/) to read from a bucket.
+
+### Step 3: AI DIAL Analytics Realtime
+
+Follow the [instructions](https://github.com/epam/ai-dial-analytics-realtime/blob/development/README.md) to setup AI DIAL Analytics Realtime service.
+
+### Step 4: Prompt Log Collector
+
+By default, AI DIAL Core uses the external open-source solution [vector.dev](https://vector.dev/) to send **chat completion logs** to storages such as AWS S3, Azure Blob Store, GCP Cloud Storage or any other "sink". It is also used to send prompt logs to AI DIAL Analytics Realtime via HTTP.
+
+> You can find more details on delivering observability data to an HTTP server in the vector.dev [documentation](https://vector.dev/docs/reference/configuration/sinks/http).
+
+As a reference on how to enable log collection, you can use AI DIAL Core [Helm chart configuration](https://github.com/epam/ai-dial-helm/blob/cb7ee62f9a7f2cedb85544c4b0eaa1e07e5a24a1/charts/dial-core/values.yaml#L482).
```

Comment:
@alexey-ban the only configuration in helm chart and this is it?

Reply:
@astsiapanay please take a look

Reply:
@sr-remsha I would prefer not to mention Helm in this context. This guide can work not only for Kubernetes.